### ISSUE TITLE
Image alt text and title can't be updated from UI when editing existing posts

### ISSUE THREAD

**Issue by RDjarbeng** [sveltia/sveltia-cms#328]
### Current behavior: 
When I save a new post and try to edit the alt text and title for the images it isn't saved.
Here is an example:
This post original has this alt text and blank title
![Image](https://github.com/user-attachments/assets/c070cb4b-4cb8-494a-b9d0-fc4ce4749d3a)

When I change the alt text and add a title, the save button is still greyed out. Maybe this means the change is not recognized.
![Image](https://github.com/user-attachments/assets/a4c96839-1d7c-4dff-af4e-34066c559e76)

If I edit the post content slightly; the text in the post -not the image, the save button is activated, but when I hit save the new alt text and title are lost and the old one is placed there instead.

### Expected behavior: 

Alt text and title should be updated correctly for edited posts. Ideally without having to modify the post content

Here's the [file](https://github.com/RDjarbeng/RDjarbeng/blob/a5497bab418a86100e002c567ed959505f794813/_posts/2025-02-19-the-imperfections-of-machine-learning.md) I was working with in my repository if needed

**Comment by kyoshino** [comment]
Will check!

**Comment by kyoshino** [comment]
Fixed in [v0.58.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.58.1) 🙇🏼
